src/pages/dashboard/panels/UpcomingAnime.tsx

```diff
@@ -28,10 +28,15 @@ const UpcomingAnime = () => {
 
   const renderDetails = (item: DashboardEpisodeDetailsType ) => {
     const airDate = moment(item.AirDate);
-    
+    let relativeTime = airDate.fromNow();
+    if (relativeTime.includes('ago'))
+      relativeTime = 'today';
```

Comment:
for this you want to use moment.relativeTimeThreshold(unit, limit) to change the threshold where is starts using hours instead days etc.. and you can set it to use 'today' for anything below 24h too.

Reply:
I can't seem to find a good alternative way using the moment.js library to get it to display "today" for the current date instead of counting in hours/minutes/seconds ago. Changing the thresholds doesn't seem to be the correct answer and their doc site isn't much help...

Also, I forgot to address the case for "tomorrow" if you're minutes or seconds from midnight.